hpargparse/hputils.py

```diff
@@ -233,15 +236,21 @@ def wrapper(*args, **kwargs):
             parser.add_argument(
                 make_option("list"),
                 action="store",
-                default=False,
+                default=None,
                 const="yaml",
                 nargs="?",
-                choices=["detail", "yaml"],
+                choices=["detail", "yaml", "json"],
```

Comment:
Because there add `--hp-list json` support, it’s best for this code to add some explain in README.md.
![image](https://user-images.githubusercontent.com/33343978/83346895-249b6c00-a353-11ea-94c4-0869b4dd7378.png)